src/rest.wiki

```diff
@@ -242,22 +240,26 @@ or a list of all existing locations otherwise.
 
 <<code language="ocaml"|
 let read_handler id_opt () =
+  let%lwt db = db in
   match id_opt with
   | None ->
     Ocsipersist.fold_step
-      (fun id loc acc -> Lwt.return ((id, loc) :: acc)) db []
-    ~>>= fun locations ->
-    let json = Yojson.to_string<locations> locations in
+      (fun id loc acc -> Lwt.return ((id, loc) :: acc)) db []  
+    >>= fun locations ->
```

Comment:
We do need the `~`. It is used to escape our Wiki syntax, which assigns meaning to `>>`. (Same issue in multiple places.)